pubsub/listener.rb

```diff
@@ -0,0 +1,32 @@
+# Copyright 2015 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "sinatra"
+require "json"
+require "base64"
+
+configure do
```

Comment:
`configure` block not required to set the port.  This can be simplified to simply:

```ruby
# ...
require "base64"

set :port, 8080

post "/push" do
  # ...
end
# ...
```

Port 8080 is only relevant to App Engine Managed VMs so I would vote to exclude it from our region snippet code, eg.

```ruby
# [START push_message_handler]
require "sinatra"
require "json"
require "base64"

post "/push" do
  # ...
end
# [END push_message_handler]

set :port, 8080
```